posts/2021-09-27-Core-team-membership-updates.md

```diff
@@ -0,0 +1,96 @@
+up of team leads, it follows that team leads were also overwhelmingly composed
+of Mozilla employees. As Rust has grown, folks previously employed at Mozilla
+left for new jobs and new folks appeared. Many of the new folks were not
+employed to work on Rust full time so the collective time investment was
+decreased and the shape of the core team’s work schedule shifted from 9-5 to a
+more volunteer cadence.  Currently, the Core team is composed largely of
+volunteers, and no member of the Core team is employed full time to work on
+their Core team duties.
+
+We know that it’s critical to driving this work successfully to have
+stakeholders on the team who are actively working in all areas of the project to
+help prioritize the Core team’s initiatives. To serve this goal, we are
+announcing some changes to the Core team’s membership today: Ryan Levick,
+Jan-Erik Rediger, and JT are joining the Core team. To give some context on
+their backgrounds and experiences, each new member has written up a brief
+introduction.
+
+* [Ryan Levick](http://github.com/rylev) began exploring Rust in 2014 always
+  looking for more and more ways to be involved in the community. Over time he
+  participated more by co-organizing the Berlin Rust meetup, doing YouTube
+  tutorials, helping with various project efforts, and more. In 2019, Ryan got
+  the opportunity to work with Rust full time leading developer advocacy for
+  Rust at Microsoft and helping build up the case for Rust as an official
+  language inside of Microsoft. Nowadays he’s an active Rust project member with
+  some of the highlights including working in the compiler perf team, running
+  the Rust annual survey, and helping the 2021 edition effort.
+* [Jan-Erik Rediger](https://github.com/badboy) started working with Rust
+  sometime in late 2014 and has been a member of the Rust Community Team since
+  2016. That same year he co-founded RustFest, one of the first conferences
+  dedicated to Rust. In the following years seven RustFest conferences have
+  brought together hundreds of Rust community members all around Europe and
+  more recently online.
+* [JT](https://github.com/jntrnr) has 15 years of programming language
+  experience. During that time, JT worked at Cray on the Chapel programming
+  language and at Apple on LLVM/Clang.  In 2012, they joined Microsoft as part
+  of the TypeScript core team, where they helped to finish and release
+  TypeScript to the world. They stayed on for over three years, helping direct
+  TypeScript and grow its community. From there, they joined Mozilla to work on
+  Rust, where they brought their experience with TypeScript to help the Rust
+  project transition from a research language to an industrial language. During
+  this time, they co-created the new Rust compiler error message format and the
+  Rust Language Server. Their most recent work is with Nushell, a programming
+  language implemented in Rust.
```

Comment:
A "programming language implemented in Rust"? That sounds potentially confusing to the reader and inaccurate to me. Officially it calls itself "A new type of shell."

Reply:
@therealprof - Nushell is both a shell and a language. Happy to chat more, though probably best to chat details on the Nushell repo/discord server.

Reply:
@jntrnr I'm aware (as a contributor), although I wouldn't quite see it as a programming language really, same as I wouldn't see SQL as a programming language. For the occasional reader this might read weird but if you're happy with that representation, that's certainly fine with me.